spec/lib/data_update_scripts/re_index_existing_articles_spec.rb

```diff
@@ -0,0 +1,8 @@
+require "rails_helper"
+require Rails.root.join("lib/data_update_scripts/20200217131245_re_index_existing_articles_with_approved.rb")
+
+describe DataUpdateScripts::ReIndexExistingArticlesWithApproved do
+  it "triggers a re-index on articles" do
+    expect(Article).to respond_to(:trigger_index)
```

Comment:
I think this is only testing if the article object has the method `trigger_index`, but it's not testing if running script triggers that method.

As you're loading article objects from the DB I don't even think there's an easy way to test this. You might want to test it like this:

```ruby
create(:article)
described_class.new.run
# check there are the proper enqueued jobs
```

Reply:
@rhymes I kept this purposefully simple after chatting with @mstruve, the most important part of the script was to test that when we remove Algolia from the Article model we would know to remove the data update. This code isn't going to be around for long and I didn't want to spend too much time on it

Reply:
makes sense!

Reply:
Thanks @rhymes!

Reply:
Would be kinda nice to just have a simple framework that runs all of the data update scripts before the test suite, kinda like migrations, and just ensures we dont error out. https://github.com/thepracticaldev/dev.to/projects/9#card-33234638